docs/migration.md

```diff
@@ -1,307 +1,14 @@
-
-- All components in this package now use native `<table>` styling.
-  - Previous implementations may have hard-coded column widths. This may no longer be necessary.
-- All state and styling props such as `active`, `focused`, ‘hover` etc. have been removed.
-- The following props have been renamed:
-  - `Cell`
-    - `minimum` -> `isMinimum`
-    - `truncate` -> `isTruncated`
-    - `menu` -> `hasOverflow`
-  - `Row`
-    - `striped` -> `isStriped`
-    - `selected` -> `isSelected`
-
-#### @zendeskgarden/react-tabs
-
-- A new structure is required for the `Tabs` component.
-  - This new structure allows consumers to style individual tags if necessary.
-
-```jsx
-<Tabs>
-  <TabList>
-    <Tab item="tab-1">Tab 1</Tab>
-  </TabList>
-  <TabPanel item="tab-1">Content</TabPanel>
-</Tabs>
-```
-
-- All state and styling props such as `active`, `focused`, `hover`, `selected`, etc. have been removed.
-- The `vertical` prop has been renamed to `isVertical`.
-
-#### @zendeskgarden/react-tags
-
-- Child component exports `Avatar` and `Close` have been removed.
-  - Use `Tag.Avatar` and `Tag.Close` instead.
-- The following props have been renamed:
-  - `Tag`
-    - `pill` -> `isPill`
-    - `round` -> `isRound`
-    - `type` -> `hue`
-
-#### @zendeskgarden/react-theming
-
-- `retrieveTheme` has been renamed to `retrieveComponentStyles`.
-- All theming is now provided through a single `theme` prop.
-  - RTL notation
-    - Previously `<ThemeProvider rtl>`
-    - Currently `<ThemeProvider theme={{ ...DEFAULT_THEME, rtl: true }}>`
-  - Custom `document` object
-    - Previously `<ThemeProvider document={doc}>`
-    - Currently `<ThemeProvider theme={{ ...DEFAULT_THEME, document: doc }}>`
-- Per component styles
-  - Previously `theme[COMPONENT_ID]`
-  - Currently `theme.components[COMPONENT_ID]`.
-
-#### @zendeskgarden/react-tooltips
-
-- The following exports have been removed:
-  - `LightTooltip`
-  - `TooltipView`
-  - `GARDEN_PLACEMENTS`
-  - `POPPER_PLACEMENTS`
-  - `getPopperPlacement`
-  - `getRtlPopperPlacement`
-- The following props have been renamed or removed:
-  - `delayMilliseconds` -> `delayMS`
-  - `arrow` -> `hasArrow`
-  - `appendToBody` -> `appendToNode`
-    - It is required to pass the HTML element which the tooltip should append to.
-  - `trigger` -> removed
-    - Tooltip `children` now accepts a single element which acts as the triggering element
-- Tooltip `content` prop now accepts Tooltip content (previous `children`)
-- The Tooltip trigger no longer has a wrapping `<div>` element
-  - An optional `refKey` prop has been added for retrieving the ref from the triggering element.
-
-#### @zendeskgarden/react-typography
-
-- The following props have been renamed:
+#### @zendeskgarden/react-utilities
 
-  - `Code`
-    - `type` -> `hue`
-  - `Typography`, `MD`, `LG`, `SM`,
-    - `monospace` -> `isMonospace`
+- This package has been removed. Migrate to `@zendeskgarden/container-utilities` to continue
+  receiving updates.
```

Comment:
Let's also call out theming utilities that exist under `@zendeskgarden/react-theming`. Between `react-theming` and `container-utilities`, consumers should be able to find what Garden has to offer.

Reply:
✅